src/EventInvocation.js

```diff
@@ -248,9 +248,15 @@ export class EventInvocation {
           const eventName = EventHandlerObject.eventName;
           const eventRegistrationID = uniqueListenerKey.split('-')[1];
           const [sdkType, module] = this.getSdkTypeAndModule(eventNameWithModuleName);
-          // going to unregister the event listener by registrationID
+          // going to unregister the event listener by eventName
```

Comment:
Add additional documentation for how listeners are cleared between SDK and Transport modes

Reply:
Addressed the comment